spinn_front_end_common/interface/abstract_spinnaker_base.py

```diff
@@ -1446,8 +1446,11 @@ def _get_machine(self, total_run_time=0.0, n_machine_time_steps=None):
                 self._graph_mapper = executor.get_item(
                     "MemoryGraphMapper")
 
-        if self._txrx is not None and self._app_id is None:
-            self._app_id = self._txrx.app_id_tracker.get_new_id()
+        if self._app_id is None:
+            if self._txrx is None:
+                self._app_id = 16
```

Comment:
Why 16? Why not our good old favourite random number, 4?

Reply:
Only @alan-stokes is allowed to use magic numbers

Reply:
Sorry, no can do. 0-15 reserved ;)

Reply:
OK